boba-chain-ops/genesis/alloc_migration_test.go

```diff
@@ -0,0 +1,60 @@
+package genesis
+
+import (
+	"encoding/json"
+	"reflect"
```

Comment:
Same note here about not using `reflect` for assertions in tests, let's prefer testify (or some other matcher like gomega if you prefer)